bodybank/character_generator/templates/ego.html

```diff
@@ -1,8 +1,4 @@
-<head>
-    <title>The Ego Page</title>
-    <h1>Ego Page!</h1>
-    <h3>Hi there, thanks for visiting the ego page!</h3>
-</head>
-<body>
-
-</body>
+<form action="/ego/" method="post">
```

Comment:
I would not hardcode urls, you can leave it empty to default to same page or use `{% url ... %}`